src/components/ComponentDocsList/ComponentDocsList.sass

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/core'
+
+.ComponentDocsList
+    margin-left: 32rem
+    padding: 15px
+    background-color: white
+    min-height: calc(100vh - $nav-height)
```

Comment:
It wasn't doing what it was supposed to do, but it didn't matter anyway. Removed.